{{cookiecutter.__src_folder_name}}/.devcontainer/devcontainer.json

```diff
@@ -25,13 +25,14 @@
                 "charliermarsh.ruff",
                 "ms-python.python",
                 "ms-python.black-formatter",
-                "microsoft.bicep",
```

Comment:
This extension does not seem to exist

Reply:
The extension id is - `ms-azuretools.vscode-bicep`. Can we add that in?